client/src/components/HistoricalComponents/historicalData/addSessionModal.js

```diff
@@ -1,97 +1,82 @@
-import React from "react";
+import React, { useState } from "react";
 import { Row, Col, Table, Button, Form, Modal } from "react-bootstrap";
+import Select from 'react-select';
 import modalStyling from './_styling/modalStyling.module.css';
 
+const subTeamOptions = [
+  { value: 1, label: 'Frame & Body' },
+  { value: 2, label: 'Suspension' },
+  { value: 3, label: 'Powertrain' },
+  { value: 4, label: 'Electrical' },
+  { value: 5, label: 'Aero' },
+];
 export default class AddSessionModal extends React.Component {
   constructor(props) {
     super(props);
     this.name = React.createRef();
-    this.subteam = React.createRef();
-
     this.state = {
       show: this.props.show,
       hide: this.props.hide,
```

Comment:
Can you remove these states? It is not used anyways. Also, it is not a good idea to use the props you get as states, there are only very little cases where you would use props as a state